queries.sql

```diff
@@ -8,3 +8,10 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true ;
 SELECT * FROM animals WHERE name IS DISTINCT FROM 'Gabumon' ;
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3 ;
+
```

Comment:
Please be sure to make the following transactions 🤓

### Inside a transaction:


UPDATE the animals table by setting the species column to unspecified, then roll back the change (**for this step students are explicitly asked to verify that change was made and rollbacked, so make sure that they included all steps as in the example below **).


### Inside a transaction:


UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.

UPDATE the animals table by setting the species column to pokemon for all animals that don't have species already set.
Commit the transaction.

### Inside a transaction delete all records in the animals table, then roll back the transaction.

### Inside a transaction:


DELETE all animals born after Jan 1st, 2022
Create a savepoint for the transaction

UPDATE all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint

UPDATE all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction

Reply:
Thank you for your feedback. I did those transactions. Am I supposed to put them in the projet ?

Reply:
Fantastic! Yes, please 🚀